### DISCUSSION TITLE
Maelstrom V0.5 bug ?

### DISCUSSION THREAD

**Discussion by PetitYaourt** [sleepeybunney/maelstrom#56] [General]
Hello i was doing a seed (1163500096) with the V0.5 on twitch and when I was suppose to fight against Mobile Type-8 the randomizer replace it by Fujin and Raijin (2nd) but there was only Fujin in the fight.

You can see the clip of the fight:
https://clips.twitch.tv/KindPiercingPancakePeanutButterJellyTime-DwWDInM8LmWUyBKs

Sorry for my bad english, this is not my native language

PS: I really love the game and more the randomizer, i'm playing so much seed, i just can't get enough. I send you my best thanks for your job :)

**Comment by sleepeybunney** [comment]
Thanks for letting me know! No idea why that happened but I will find out
